Source/Game.cs

```diff
@@ -118,6 +117,11 @@ public Game()
 		imGuiManager = new ImGuiManager();
 	}
 
+	public string GetFullVersionString()
+	{
+		return $"{VersionString} + {LoaderVersion}";
+	}
```

Comment:
We maybe should split this onto two lines for consistency with the other places we show the version strings, and just to make it easier to read.

Reply:
Changed in b75a99e